lxd/storage_volumes_snapshot.go

```diff
@@ -122,7 +123,7 @@ func storagePoolVolumeSnapshotsTypePost(d *Daemon, r *http.Request) response.Res
 			return err
 		}
 
-		return pool.CreateCustomVolumeSnapshot(projectName, volumeName, req.Name, op)
+		return pool.CreateCustomVolumeSnapshot(projectName, volumeName, req.Name, time.Unix(0, 0), op)
```

Comment:
@monstermunchkin you've used `time.Unix(0,0)` here and `time.Time{}` in other places, is there a difference? Maybe worth defining a constant in the `storage` package called `ExpiryNone` or something like that you can use across the board.

Reply:
Yes, there's a difference but no reason for not sticking to one of them. I'll change those to `time.Time{}`. `time.Unix(0, 0)` is 1970 something something, and `time.Time{}` is just empty.